azure-stack/hci/deploy/deployment-prep-active-directory.md

```diff
@@ -56,6 +56,13 @@ The *AsHciADArtifactsPreCreationTool.ps1* module is used to prepare Active Direc
 |`-AsHciDeploymentPrefix`|The prefix used for all AD objects created for the Azure Stack HCI deployment. <br> The prefix must not exceed 8 characters.|
 |`-Deploy`|Select this scenario for a brand new deployment instead of an upgrade of an existing system.|-->
 
+> [!NOTE]
+> The `-AsHciOUName` path does not support the following special characters **anywhere** within the OUPath.
+> 
```

Comment:
Change: The `-AsHciOUName` path does not support the following special characters **anywhere** within the OUPath.
**& < > \ '**

To: 
> - The `-AsHciOUName` path doesn't support the following special characters anywhere within the path - `&,”,’,<,>`.
> - Moving the computer objects to a different OU once the deployment is complete, is also not supported.